src/components/combobox/combobox.scss

```diff
@@ -143,6 +143,7 @@
 }
 
 .floating-ui-container {
+  --calcite-floating-ui-z-index: theme("zIndex.dropdown");
```

Comment:
Nitpick: Should we rename `zIndex.dropdown` to `zIndex.default` since it is now being used as the default, and no longer specific to dropdown?

Reply:
hmm our tailwind already has a zindex.default of 1. I think this is fine as is.